routes/organizations.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var router = express.Router();
+const {Organizations} = require('../models')
+
+/* GET organizations listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with a resource');
+  
+});
+
+router.get('/1/public', async function(req, res) {
+  const listOfOrganizations = await Organizations.findAll();
+  const {name, image, phone, address, welcomeText} = listOfOrganizations;
```

Comment:
listOfOrganizations, que tipo de dato es?

Reply:
👍  al crear las evidencias noté que estaba obteniendo una lista con todas las filas de la tabla. Ahora la peticion recibe un id en reemplazo de ese "1" constante. Dicho ID hacer referencia al ID de la tabla testimonials. De esta manera podemos hacer una busqueda por ID lo cual nos devuelve tan solo un objeto con la fila especificada y podemos realizar el destructuring de manera correcta.